src/app/components/create-category/create-category.component.html

```diff
@@ -0,0 +1,35 @@
+<form
+  [formGroup]="createCategoryForm"
+  (ngSubmit)="onCreateCategoryFormSubmitted(createCategoryForm)"
+>
+  <div>
+    <mat-form-field appearance="outline" class="mb-5"
+      ><mat-label>Name</mat-label
+      ><input
+        matInput
+        type="text"
+        placeholder="Name"
+        formControlName="name"
+        required
+      />
+      <!--  TODO: Add proper validation  -->
+      <mat-error *ngIf="!createCategoryForm.get('name')!.errors?.required"
+        >Please use letters only</mat-error
+      >
+      <mat-error *ngIf="!createCategoryForm.get('name')!.errors?.pattern"
+        >Entering a name is obligatory</mat-error
+      >
+    </mat-form-field>
+  </div>
+  <div>
+    <!-- https://itnext.io/valid-and-invalid-in-angular-forms-61cfa3f2a0cd -->
```

Comment:
yes valid is better, but we don't need article here :D

Reply:
This is for me more than anyone else I think, I will make this one a proper repository (not just a fork), and keep it for the future. Always good to know why I put valid there, a year ago I would not even think about using anything else than 'invalid'